### PR TITLE
 NXP-20773 &  NXP-22588: user cloud services and authorized apps

### PR DESCRIPTION
**Pull request by Gabez0r**
Depends on nuxeo/nuxeo#1888.

### CODE REVIEW

File: elements/nuxeo-cloud-services/nuxeo-user-authorized-apps.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+<link rel="import" href="nuxeo-oauth2-tokens.html">
+
+<!--
+`nuxeo-user-authorized-apps`
+@group Nuxeo UI
+@element nuxeo-user-authorized-apps
+-->
+<dom-module id="nuxeo-user-authorized-apps">
+  <template>
+    <style include="nuxeo-styles iron-flex iron-flex-alignment">
```
Comment (afjusto) [inline elements/nuxeo-cloud-services/nuxeo-user-authorized-apps.html]: maybe you can drop the `iron-flex-alignment`?

File: elements/nuxeo-cloud-services/nuxeo-user-cloud-services.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+<link rel="import" href="nuxeo-oauth2-tokens.html">
+
+<!--
+`nuxeo-user-cloud-services`
+@group Nuxeo UI
+@element nuxeo-user-cloud-services
+-->
+<dom-module id="nuxeo-user-cloud-services">
+  <template>
+    <style include="nuxeo-styles iron-flex iron-flex-alignment">
```
Comment (afjusto) [inline elements/nuxeo-cloud-services/nuxeo-user-cloud-services.html]: maybe you can drop the `iron-flex-alignment`?

**Comment by afjusto** [comment]
👍

File: elements/nuxeo-cloud-services/nuxeo-cloud-services.html [hunk 3]
```diff
@@ -31,10 +31,7 @@
 -->
 <dom-module id="nuxeo-cloud-services">
   <template>
-    <style>
-      .flex {
-        @apply --layout-flex;
-      }
+    <style include="nuxeo-styles iron-flex iron-flex-alignment">
```
[the remark this answers is not in the document]
Comment (Gabez0r) [reply elements/nuxeo-cloud-services/nuxeo-cloud-services.html]: We need `iron-flex` for the `flex` class, but we don't need `iron-flex-alignment` here.

File: elements/nuxeo-cloud-services/nuxeo-oauth2-tokens.html [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+        <template>
+          <paper-checkbox noink checked="[[item.isShared]]" disabled></paper-checkbox>
+        </template>
+      </nuxeo-data-table-column>
+
+      <nuxeo-data-table-column name="[[i18n(col.name)]]" key="[[col.key]]">
+        <template>
+          <paper-icon-button id="edit-button-[[index]]" name="edit" icon="nuxeo:edit"
+                              on-tap="_editEntry"></paper-icon-button>
+          <paper-tooltip for="edit-button-[[index]]">[[i18n('cloudTokens.edit')]]</paper-tooltip>
```
[the remark this answers is not in the document]
Comment (Gabez0r) [reply elements/nuxeo-cloud-services/nuxeo-oauth2-tokens.html]: That's what we already have on `nuxeo-cloud-tokens`.

**Pull request merged**